src/js/review/SubmitController.jsx

```diff
@@ -89,21 +103,21 @@ class SubmitController extends React.Component {
       form,
       formConfig,
       showPreSubmitError,
-      renderErrorMessage,
-      submission
+      renderErrorMessage
     } = this.props;
+    const preSubmit = this.getPreSubmit(formConfig);
+
     return (
       <div>
-        { this.preSubmitInfo && <PreSubmitSection
-          required
-          preSubmitInfo={formConfig.preSubmitInfo}
-          onChange={() => this.props.setPreSubmit(formConfig.preSubmitInfo.field, this.value)}
-          form={form}
+        <PreSubmitSection
```

Comment:
So now that we're always initializing `preSubmit` to _something_ through the `getPreSubmit` thing at the top of this file, we don't need to check whether or not to render this component based on what's in the `formConfig`? Because we initialize it to `false`, that will mean if there's nothing else specified in the `formConfig` nothing will actually be rendered for the checkbox?

Reply:
Right, if nothing is specified we'll have a `notice` that is empty (so it renders nothing) and a `required` that is not `true` (it's `undefined`) so the checkbox and label won't render.